github/AuthenticatedUser.py

```diff
@@ -177,7 +177,7 @@ def _initAttributes(self) -> None:
         self._two_factor_authentication: Attribute[bool] = NotSet
 
     def __repr__(self) -> str:
-        return self.get__repr__({"login": self._login.value})
+        return self.get__repr__({"login": self._login.value or "<UNKNOWN>"})
```

Comment:
I'd prefer `(UNKNOWN)`.
```suggestion
        return self.get__repr__({"login": self._login.value or "(UNKNOWN)"})
```